packages/@okta/vuepress-site/guides/protect-your-api/configure-cors/springboot/configcors.md

```diff
@@ -1,4 +1,23 @@
-You can enable CORS with the `CrossOrigin` annotation in your controller. For example:
+In order to configure CORS in Spring Security you must enable it in the `WebSecurityConfigurerAdapter` you defined in the previous step:
+
+```java
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+class OktaOAuth2WebSecurityConfigurerAdapter extends WebSecurityConfigurerAdapter {
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        // previous configuration
+        ...
+        http.cors();
```

Comment:
@nbarbettini @wje600 
This could _easily_ get moved the previous page, but we don't talk about CORS over there?  Is this clear enough?

Reply:
Clear to me 👍

Reply:
I like keeping the CORS stuff all together for clarity.